.github/workflows/nightly-trigger.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       matrix:
         branch:
           - master
+          - release-1.20
           - release-1.19
-          - release-1.18
```

Comment:
Don't remove the 1.18 release branch just, yet. There might be another 1.18.2 being released before 1.18 finally reaching end-of-life.